src/rust/engine/fs/brfs/Cargo.toml

```diff
@@ -6,7 +6,7 @@ authors = ["Pants Build <pantsbuild@gmail.com>"]
 publish = false
 
 [dependencies]
-clap = { workspace = true }
+clap = { workspace = true, features = ["string"] }
```

Comment:
The "string" feature allows `default_value` to take dynamic strings, which we use in a couple of places. Otherwise they must be `&'static str`...